src/test/java/L2Z5/EvenOrOddTest.java

```diff
@@ -0,0 +1,30 @@
+package L2Z5;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class EvenOrOddTest {
+    @Test
+    void shouldTrueIfNumberIsEven() {
+        //given
+        int number = 4;
+
+        //when
+        boolean x = EvenOrOdd.checkIfEven(number);
+
+        //then
+        Assertions.assertTrue(x);
+    }
+
+    @Test
+    void shouldTrueIfNumberIsOdd() {
```

Comment:
'shouldTrueIfNumberIsOdd'

a na dole 

`   Assertions.assertFalse(x);`
to w końcu shouldTrue czy jednak nie do końca shouldTrue? ;)